bundles/org.openhab.binding.playstation/src/main/java/org/openhab/binding/playstation/internal/PS3Handler.java

```diff
@@ -50,6 +50,7 @@
 
     private final Logger logger = LoggerFactory.getLogger(PS3Handler.class);
     private static final int SOCKET_TIMEOUT_SECONDS = 2;
+    private boolean isDisposed = false;
```

Comment:
This should be set to false during initialization. Otherwise the handler instance can't be reused.